package.json

```diff
@@ -32,25 +32,27 @@
   },
   "dependencies": {
     "asar": "^0.13.0",
-    "async": "^2.0.0",
-    "debug": "^2.2.0",
-    "fs-extra": "^1.0.0",
+    "async": "^2.5.0",
+    "debug": "^3.1.0",
+    "fs-extra": "^4.0.2",
```

Comment:
This update for certain raises the minimum Node version to 4. Probably others do the same thing. I'm perfectly OK with that (I dropped Node < 4 support in Packager a while ago), and it might have already done it when Node 6 support was added, but in my experience we might need to note it in the README.

Reply:
Good point, feel free to change the README